tests/translator/input/api_with_auth_all_minimum_openapi.yaml

```diff
@@ -0,0 +1,81 @@
+Globals:
```

Comment:
@brettstack I could deploy this template (with valid s3 bucket). @keetonian This template should cover tests for #2 and #3 in your comment.